src/actions/validators.js

```diff
@@ -0,0 +1,27 @@
+import { Jurisdiction } from '../contracts'
+import FetchingActions from './fetching'
+import * as ActionTypes from '../actiontypes'
+import { JURISDICTION_ADDRESS } from '../constants'
+import Network from "../network";
+
+const ValidatorsActions = {
+
+  findAll() {
+    return async function(dispatch) {
+      const jurisdiction = await Jurisdiction.at(JURISDICTION_ADDRESS)
+      const events = jurisdiction.ValidatorAdded({}, { fromBlock: 0, toBlock: 'latest' });
+      events.watch(function(error, result) {
+        if(error) throw error
```

Comment:
Let's add an `XXX` note here or something to make sure we do proper error handling later on.

Reply:
No, the throw is not enough. When we were pairing, we left the alerts for later. But now I think it's a good time to copy them.